server/api/reports.go

```diff
@@ -65,15 +62,15 @@ type TestResult struct {
 	// specified by the user, used for querying
 	Duration time.Duration `json:"duration"`
 	// actual Elapsed time, used for X-per-second kind of metrics
-	Elapsed time.Duration `json:"elapsed"`
-	// events per request
-	Events int `json:"events"`
+	Elapsed      time.Duration `json:"elapsed"`
+	Errors       int           `json:"errors"`
```

Comment:
wdyt about having up there in the metadata section (~L30) and `APIVersion` attribute hardcoded to `v2`?
we could then filter for it in kibana dashboards to separate v2 form v1.

not important thou, we can as well start from scratch

Reply:
Would you mind creating an issue for this? I see this a general v2 add-on and would like to move on with this PR.

Reply:
agreed, will do 👍